stdlib/3.5/pathlib.pyi

```diff
@@ -64,20 +65,32 @@ class Path(PurePath):
     def owner(self) -> str: ...
     def read_bytes(self) -> bytes: ...
     def read_text(self, encoding: Optional[str] = ...,
-                  errors: Optional[str] = ...) -> bytes: ...
-    def rename(self, target: Union[str, Path]) -> None: ...
-    def replace(self, target: Union[str, Path]) -> None: ...
+                  errors: Optional[str] = ...) -> str: ...
+    def rename(self, target: Union[str, PurePath]) -> None: ...
+    def replace(self, target: Union[str, PurePath]) -> None: ...
     def resolve(self) -> Path: ...
     def rglob(self, pattern: str) -> Generator[Path, None, None]: ...
     def rmdir(self) -> None: ...
-    def samefile(self, other_path: Union[str, Path]) -> bool: ...
-    def symlink_to(self, target: Union[str, Path],
+    def samefile(self, other_path: Union[AnyStr, int, Path]) -> bool: ...
```

Comment:
There are some bugs around using the `AnyStr` type variable in only one place.  Could you write this as `Union[str, bytes, int, Path]` instead?